packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -40,6 +40,17 @@ const cli = {
           return [];
         },
       },
+      buildBranch: {
```

Comment:
Now that we have this added here, can we update the [function documentations in test-runner.js](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/terra-cli/wdio/test-runner.js#L33-L53) to include this option for consistency? It would be similar to this [change](https://github.com/cerner/terra-toolkit/pull/732/files#diff-f13ec26e86b6e8d765ec9020014d38600b7df8643e37a26d40000d28807e2bfb).

Reply:
Updated here https://github.com/cerner/terra-toolkit/pull/736/commits/dd9fbb2487069446ec10a8e4109fd6729ab9599e#diff-f13ec26e86b6e8d765ec9020014d38600b7df8643e37a26d40000d28807e2bfbR36